packages/components/table/src/store/watcher.ts

```diff
@@ -137,7 +137,7 @@ function useWatcher<T>() {
 
   // 选择
   const isSelected = (row) => {
-    return selection.value.includes(row)
+    return selection.value.some((item) => item.id === row.id)
```

Comment:
Does this `id` attribute always exist?

Reply:
good question, I have replaced `id` with `row-key`

Reply:
Keep the original code does not work properly?